### ISSUE TITLE
Move hot reload gif to a youtube video

### ISSUE THREAD

**Issue by mit-mit** [flutter/flutter#14139]
We have a large animated gif file illustrating hot reload: `/images/intellij/hot-reload.gif`

This single file accounts for the majority of our website traffic (in bytes). We should consider replacing this with a video on Youtube.

**Comment by InMatrix** [comment]
Can we consider using a HTML5 video instead of a gif or a Youtube video? According to this article, the size of HTML5 video is considerably smaller than gifs: https://rigor.com/blog/2015/12/optimizing-animated-gifs-with-html5-video

The side effect of hosting videos on Youtube is that users in China won't be able to see them unless they use a VPN.

**Comment by xster** [comment]
A streamable.com embed with autoplay might be more lightweight.

**Comment by sethladd** [comment]
Or, just a `<video>` tag.

**Comment by filiph** [comment]
Alright, I played around with this today. The change will go live soon, as a part of another CL.

TL;DR: We're staying with gif, but the gif is now 693KB instead of 6MB (thanks to some `ffmpeg`/`convert` magic). It still looks really good:

![increment_hot_reload_small](https://user-images.githubusercontent.com/919717/36707363-d090851a-1b22-11e8-8e15-8eb90fa4d6d7.gif)

We could get even higher framerate from an similarly sized mp4 / webm, but:

* Not considerably higher (60 fps versus the gif's 20 fps is definitely higher, but it's not as a huge difference as with 12fps vs 20 fps we're getting now, imho).
* My testing showed that Safari just doesn't show the video. I'm sure there's a workaround.
* If we provide the `.gif` as a fallback, Chrome (and probably other browsers) will download _both_ the video _and_ the fallback.

For posterity, here's the `ffmpeg`/`convert` magic:

```
$ ffmpeg -i input.mov -s 878x522 -r 20 -f image2pipe -vcodec ppm - >> temp.XXX
$ cat temp.XXX | convert +dither -layers Optimize -loop 0 -delay 5 - output.gif
```

I'm closing this issue for now. We can revisit.

cc @mjohnsullivan

**Comment by xster** [comment]
super mini nit: I think the active window drop shadows are clipped
